vspace/src/main/java/edu/asu/diging/vspace/core/model/display/IExternalLinkDisplay.java

```diff
@@ -11,4 +11,8 @@
     void setExternalLink(IExternalLink link);
 
     IExternalLink getExternalLink();
+    
+    void setType(DisplayType type);
+
+    DisplayType getType();
```

Comment:
Since SpaceLinkDisplays have a displayType too, let's put that in the shared superclass.